src/libraries/System.Private.CoreLib/src/System/IO/Path.Unix.cs

```diff
@@ -9,9 +9,36 @@ namespace System.IO
 {
     public static partial class Path
     {
-        public static char[] GetInvalidFileNameChars() => new char[] { '\0', '/' };
+        public static char[] GetInvalidFileNameChars() => new char[] {'\0', '/'};
```

Comment:
nit: please keep the spaces

```suggestion
        public static char[] GetInvalidFileNameChars() => new char[] { '\0', '/' };
```

Reply:
Done